index.html

```diff
@@ -797,6 +797,58 @@ <h2 id="BaseAudioContext">
             </p>
           </dd>
           <dt>
+            readonly attribute DOMHighResTimeStamp currentPerformanceTime
+          </dt>
+          <dd>
+            <p>
+              The <code>DOMHighResTimeStamp</code> representation of <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>.
+            </p>
+            <p>
+              The <a href=
+              "#widl-BaseAudioContext-currentPerformanceTime"><code>currentPerformanceTime</code></a>
+              property contains the <code>DOMHighResTimeStamp</code>
+              representing the time in milliseconds from the <code>time
+              origin</code> (described in [[!hr-time-2]]) to the moment when
```

Comment:
`DOMHighResTimeStamp` is a time in microsecond, with a granularity of 5 microseconds.